generator/generator_tests/altitude_test.cpp

```diff
@@ -139,12 +147,16 @@ void TestAltitudes(MwmValue const & mwmValue, string const & mwmPath,
     TEST_EQUAL(altitudes.size(), pointsCount, ());
 
     for (size_t i = 0; i < pointsCount; ++i)
-      TEST_EQUAL(altitudeGetter.GetAltitude(f.GetPoint(i)), altitudes[i], ("A wrong altitude"));
+    {
+      TAltitude const fromGetter = altitudeGetter.GetAltitude(f.GetPoint(i));
+      TAltitude const expected = (fromGetter == kInvalidAltitude ? kDefaultAltitudeMeters : fromGetter);
```

Comment:
I didn't got your idea. The idea of the test is to form mwm with altitude section with the help of a test (mock) altitude getter and then to read features from the mwm and compare with altitude from getter. Could you please clarify your idea?